src/overlays/actors/ovl_Dm_An/z_dm_an.c

```diff
@@ -190,34 +220,42 @@ Actor* func_80C1C8E8(PlayState* play) {
     return foundActor;
 }
 
-void func_80C1C958(DmAn* this, PlayState* play) {
+void DmAn_Initialize(DmAn* this, PlayState* play) {
     if ((this->an4ObjectSlot > OBJECT_SLOT_NONE) && SubS_IsObjectLoaded(this->an4ObjectSlot, play) &&
         (this->msmoObjectSlot > OBJECT_SLOT_NONE) && SubS_IsObjectLoaded(this->msmoObjectSlot, play)) {
         ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 14.0f);
-        SkelAnime_InitFlex(play, &this->skelAnime, &object_an1_Skel_012618, NULL, this->jointTable, this->morphTable,
-                           OBJECT_AN1_LIMB_MAX);
+        SkelAnime_InitFlex(play, &this->skelAnime, &gAnju1Skel, NULL, this->jointTable, this->morphTable,
+                           ANJU1_LIMB_MAX);
 
         this->animIndex = DMAN_ANIM_NONE;
-        DmAn_ChangeAnim(this, play, DMAN_ANIM_0);
+        DmAn_ChangeAnim(this, play, DMAN_ANIM_SIT_IN_DISBELIEVE);
         this->actor.flags &= ~ACTOR_FLAG_TARGETABLE;
         Actor_SetScale(&this->actor, 0.01f);
-        this->unk_2AE |= 1;
-        this->actor.draw = func_80C1D0B0;
+        this->stateFlags |= DMAN_STATE_LOST_ATTENTION;
+        this->actor.draw = DmAn_Draw;
 
         if ((play->sceneId == SCENE_YADOYA) && (play->curSpawn == 4)) {
-            this->unk_2B4 = func_80C1C8E8(play);
-            DmAn_ChangeAnim(this, play, DMAN_ANIM_1);
+            this->lookAtActor = DmAn_FindAnjusMotherActor(play);
+            DmAn_ChangeAnim(this, play, DMAN_ANIM_SIT);
             this->actionFunc = DmAn_DoNothing;
         } else {
-            this->actionFunc = DmAn_HandleCutscene;
+            this->actionFunc = DmAn_HandleCouplesMaskCs;
         }
     }
 }
 
-void DmAn_HandleCutscene(DmAn* this, PlayState* play) {
+void DmAn_HandleCouplesMaskCs(DmAn* this, PlayState* play) {
     s32 csAnimIndex[] = {
-        DMAN_ANIM_0, DMAN_ANIM_0, DMAN_ANIM_12, DMAN_ANIM_2,  DMAN_ANIM_4,
-        DMAN_ANIM_6, DMAN_ANIM_8, DMAN_ANIM_10, DMAN_ANIM_11, DMAN_ANIM_3,
+        /* 0 */ 0, // no cue
+        /* 1 */ DMAN_ANIM_SIT_IN_DISBELIEVE,
+        /* 2 */ DMAN_ANIM_LOOK_UP,
+        /* 3 */ DMAN_ANIM_MASK_STAND_LOOP,
+        /* 4 */ DMAN_ANIM_MASK_KNEEL,
+        /* 5 */ DMAN_ANIM_HUG,
+        /* 6 */ DMAN_ANIM_HUG_SEPARATE,
```

Comment:
This is a tiny nitpick, feel free to ignore, but my brain is reading the wrong word here.
I assume this was verb "to separate" as in, to break apart, to move apart, to undo a union, and not adj "separate" as in to be apart, distinct, currently unrelated. 

Not sure if there is a better word/name or if its just me and not a real issue.

Reply:
An alternative to this animation was "unhug". Do you think it is clearer that way?

Reply:
unhug sounds like a tiktok euphemism, How about HUG_RELEASE or HUG_END?